products/pnpm.md

```diff
@@ -0,0 +1,57 @@
+---
+title: Pnpm
```

Comment:
```suggestion
title: pnpm
```

From what I can see on https://pnpm.io/.

Reply:
I am sorry, I dont see the difference?

Reply:
I think I can take this.